server/graphql_schemas/utils/helpers.py

```diff
@@ -213,5 +213,24 @@ def add_event_to_calendar(andela_user, event):
                  for attendee in event.attendees]
     calendar = build('calendar', 'v3', credentials=andela_user.credential)
     event_details = build_event(event, attendees)
-    return calendar.events().insert(
+    created_event = calendar.events().insert(
             calendarId='primary', body=event_details).execute()
+    event.event_id_in_calendar = created_event['id']
+    event.save()
+
+
+def update_event_status_on_calendar(andela_user, event):
+    eventId = event.event_id_in_calendar
```

Comment:
Kindly use snake_case for consistency. 
```suggestion
    event_id = event.event_id_in_calendar
```

Reply:
feedback implemented